cmd/backup/notifications.go

```diff
@@ -106,3 +108,12 @@ func formatBytes(b uint64, decimal bool) string {
 	}
 	return fmt.Sprintf(format, float64(b)/float64(div), "kMGTPE"[exp])
 }
+
+func toJSON(v interface{}) string {
```

Comment:
Looking at what [package sprig](https://masterminds.github.io/sprig/) does, I wonder if we should have `toJSON` (plain) and `toPrettyJSON` (indented) so that users that require a single line for whatever reason have their needs catered as well.

Reply:
If the user wants JSON, they don't care if it's one line or not. Since the user will work with the data after the JSON parser.

Reply:
I'm mostly thinking about the usecase of outputting NDJSON and then feeding this elswhere.

E.g. you could have a template like this

```
{"Type": "local", "Data": {{ .Stats.Storages.Local | toJSON }} }
{"Type": "s3", "Data": {{ .Stats.Storages.S3 | toJSON }} }
```
and get one line per storage. If the JSON output is always pretty printed, this use case wouldn't work as you'd get more than one line per item.

Reply:
I can only see 2 uses for it:
1) Template only - `{{ . | toJSON }}`
2) JSON to JSON - `{"Type": "s3", "Data": "{{ .Stats.Storages.S3 | toJSON }}" }`

That said, I think trying to create JSON based on a template is a very rare case study.

But perhaps you as the creator of the library know better where to develop the product. But then again, I don't think it's necessary here.

Reply:
I think it'd make sense to provide both pretty printed as well as standard json methods, so if you could still add this to your changes, it'd be much appreciated.

Reply:
Made 2 separate functions: `toJSON` and `toPrettyJson`.